### PR TITLE
nitrpg: introduce `loader` a tool to import git history from github API

### PR DESCRIPTION
**Pull request by Morriar**
Active demo: http://nitlanguage.org/rpg_test/games/nitlang/nit

The data displayed on this servet are loading right now.

### CODE REVIEW

File: contrib/nitrpg/src/events.nit [hunk 1]
```diff
@@ -21,6 +21,99 @@
...
+			return
+		end
+		for reactor in reactors do
+			message(3, "Apply reactor {reactor} on {event}")
+			reactor.react_event(game, event)
+		end
+		game.add_github_event new KnownGithubEvent(self, event)
+	end
+
+	# Get an event gy its github id.
```
Comment (privat) [inline contrib/nitrpg/src/events.nit]: gy?

**Comment by privat** [comment]
impressive. so much code. +1

**Comment by Morriar** [comment]
Reroll with the missing signed-off.

**Comment by privat** [comment]
ping?

**Pull request closed without merging**